recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+    - boost 1.62.*
+    - zlib 1.2.*
```

Comment:
Can you add `python` for `win` here so that this recipe would be built against all the vc versions? You also need to add vc features.
```yaml
build:
  features:
    - vc9   # [win and py27]
    - vc10  # [win and py34]
    - vc14  # [win and py35]

requirements:
  build:
    - python  # [win]
    - vc 9    # [win and py27]
    - vc 10   # [win and py34]
    - vc 14   # [win and py35]
  run:
    - vc 9   # [win and py27]
    - vc 10  # [win and py34]
    - vc 14  # [win and py35]
```

See https://github.com/conda-forge/staged-recipes/wiki/VC-features